.circleci/config.yml

```diff
@@ -32,54 +32,88 @@
             POSTGRES_PASSWORD: ""
         - image: circleci/redis:3.2-alpine # 3.2.4 engine on AWS Elasticache
 
+  defaults: &defaults
+    working_directory: ~/vets-api
+    executor: vets_api_docker_executor
+
   jobs:
     verify_pacts:
-      working_directory: ~/vets-api
-      executor: vets_api_docker_executor
+      <<: *defaults
       steps:
         - checkout_and_install
         - setup_test_db
         - verify_stable_pacts
         - store_test_results:
             path: ./rspec
+        - store_artifacts:
+            path: ./ci
+        - store_artifacts:
```

Comment:
can this key be duplicated

Reply:
Yes, there's no limit on the number of store_artifacts steps a job can run. Actually in this case, I just cleaned up a couple of the jobs that don't store any artifacts at all. Thanks for pointing this out!